lang/en/cms.php

```diff
@@ -60,8 +60,13 @@
 $string['some_name'] = 'Some name';
 $string['import_file'] = 'Import file';
 $string['import_cms_type'] = 'Import content type';
+$string['datasources'] = 'Datasources';
+$string['datasources_desc'] = 'This is a performance measure. Select datasources to be included in this type. Only selected datasources will be configurable,
```

Comment:
I know I put this in the spec, but being my own devils advocate do you think it is viable if each data source could infer if it was used so we didn't need to select it?

Going a step further do you think this would be true for potential future sources?

Reply:
I would like to make that a separate issue. I don't think it would be trivial.